esy-build

```diff
@@ -21,4 +21,4 @@ case "$(uname -s)" in
 esac
 
 # Common build steps
-make install
+exec make install
```

Comment:
Maybe we can collapse invocation of make so for windows: `exec make -j4 world.opt flexlink.opt install` Makefile is parsed once we can have better parallelism between two first targets. This allow to avoid 2 more process creations.
Also `-j4` maybe replace with `-j$NUMBER_OF_PROCESSORS`